### ISSUE TITLE
Share only the top 3

### ISSUE THREAD

**Issue by JulesMichael** [francoismari/elyze#13]
Currently, when we try to share our results, the app generate an image of the entire list of candidate with given percentage. Maybe in a goal of promoting the app, only the top 3 should be shared. So people will be more likely to share their results on social media for exemple.

**Comment by Gouvernathor** [comment]
The top 3 can easily be shared with a screenshot.
Given that the complete list cannot so easily be shared, especially with devices not including a scrolling-screenshot feature, I believe this should remain the default, if not the only, sharing option.

**Comment by JulesMichael** [comment]
> The top 3 can easily be shared with a screenshot.
> 
> Given that the complete list cannot so easily be shared, especially with devices not including a scrolling-screenshot feature, I believe this should remain the default, if not the only, sharing option.

I understand your argument, maybe instead of replacing with the top 3, there is a way to summarize this very long screenshot. For exemple the sentence «D'accord avec X% de ses propositions» is repeated for every candidate. Removing this sentence and highlight the percentage and the ranking seem to be a good way of reducing the hight of the image.

**Comment by Gouvernathor** [comment]
Yes, making the sharing thing something other than essentialy a screenshot.
Politiscales did something like that, a condensed version of the results, as you can see [in my old fork](https://gouvernathor.github.io/fr_FR/quiz/) (click the "random result" button for a sample).
